README.md

```diff
@@ -1,22 +1,164 @@
-# Working with TypeScript
+# Working with TypeScript Code Along
 
 ## Learning Goals
 
-- a
-- b
-- c
+- Successfully set up a local environment to run TypeScript
+- Configure and understand the tsconfig.json file
+- Practice compiling with `tsc`
 
 ## Introduction
 
-## a
+TypeScript is a very useful and powerful tool we can use to optimize our code by
+checking for syntax errors before runtime, giving tips on how to fix bugs before
+they happen and more, all by adding in types to our regular JavaScript code!
+Though we can start using TypeScript right out of the box, it does recognize
+that every application and developer is different, and gives us some room for
+flexibility. We get access to this flexibility/customization through the
+ts.config file that we will keep in the root of our application, where we can
+change default rules, and add some rules of our own!
 
-## b
+## Environment Setup
 
-## c
+- fork and clone this lab, then open in your text editor
+- install ts globally
+
+You should already have Node Installed on your machine, but if you haven't,
```

Comment:
Suggest changing to "... but if you don't"

Reply:
```suggestion
but if you don't
```